megatron/megatron/training/initialize.py

```diff
@@ -274,7 +282,7 @@ def _initialize_distributed(get_embedding_ranks, get_position_embedding_ranks):
     # data-parallel communicators.
     if device_count > 0:
         # Set the parallel context.
-        if args.enable_hetero:
```

Comment:
So we can remove the `args.enbale_hetero` from the code and use `args.hetero_process_meshes` instead?

Reply:
I think that the way using `process_meshes` to configure the training plan is not only designed for `heterogeneous training`, but also for other multi-model training like `Llava`. So we should unbind the `args.enbale_hetero` parameter and the `hetero_process_meshes` parameter gradually.